config/trackers.go

```diff
@@ -13,3 +13,8 @@ var Trackers = []string{
 	"udp://tracker.internetwarriors.net:1337/announce",
 	"http://mgtracker.org:6969/announce",
 	"http://tracker.baka-sub.cf/announce"}
+
+// NeededTrackers : Array indexes of Trackers for needed tracker in a torrent file
```

Comment:
But if they are not in a torrent, it doesn't show an error, it just adds it to the existing trackers of the torrent.